### DISCUSSION TITLE
Data protection when using CBC solver and Pyomo

### DISCUSSION THREAD

**Discussion by mgl204** [coin-or/Cbc#654] [Q&A]
Hi there!

I am currently working with CBC via pyomo. I installed pyomo by using the command „conda install -c conda-forge pyomo“ and CBC by using the command „conda install -c conda-forge coincbc“. I was wondering how my data is processed in relation to data protection. Is the problem solved locally on my pc or is the data sent to the solver (or any "remote" server)?

Thanks for any feedback on this!

**Comment by tkralphs** [comment]
The problem is solved locally on your PC. There is no transfer of data.
